.github/workflows/ros1.yml

```diff
@@ -21,17 +21,13 @@ jobs:
     if: ${{ github.event_name != 'push' || github.ref != 'refs/heads/ros2' }}
     name: 'Build and Bundle (ROS1)'
     container:
-      image: rostooling/setup-ros-docker:ubuntu-${{ matrix.ubuntu_distro }}-ros-${{ matrix.distro }}-ros-base-latest
+      image: ubuntu:${{ matrix.ubuntu_distro }}
     outputs:
       robot_ws_build_result: ${{ steps.robot_ws_build.outcome }}
       simulation_ws_build_result: ${{ steps.simulation_ws_build.outcome }}
     steps:
     - name: Checkout Branch
       uses: actions/checkout@v1
-    - name: Setup Permissions
-      run: |
-        # TODO(ros-tooling/setup-ros-docker#7): calling chown is necessary for now
-        sudo chown -R rosbuild:rosbuild "$HOME" .
     - name: Scan using git-secrets
       uses: aws-robotics/aws-robomaker-github-actions/git-secrets-scan-action@2.4.2
     - id: robot_ws_build
```

Comment:
[gazebo-version](https://github.com/aws-robotics/aws-robomaker-sample-application-helloworld/pull/133/files#diff-f7f0adde35fcc06134bdbef5a43bb7eb9bc44110edaf5148776308e36e3f5a4cR38) is not required anymore in the action parameters.

Valid for both `robot_ws_build`, `simulation_ws_build`